native/rust/pom.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
these changes are generated by the archetype plugin, but are unfortunately not wanted. You can simply remove this change.